pkgs/tools/misc/unicode/default.nix

```diff
@@ -1,26 +1,31 @@
-{ stdenv, fetchFromGitHub, fetchurl, python3Packages }:
+{ stdenv, fetchFromGitHub, fetchurl, python3Packages, installShellFiles }:
 
 python3Packages.buildPythonApplication rec {
   pname = "unicode";
-  version = "2.6";
+  version = "2.7";
 
   src = fetchFromGitHub {
     owner = "garabik";
     repo = "unicode";
     rev = "v${version}";
-    sha256 = "17hh4nwl5njsh7lnff583j2axn6rfvfbiqwp72n7vcsgkiszw4kg";
+    sha256 = "15d9yvarxsiy0whx1mxzsjnnkrjdm3ga4qv2yy398mk0jh763q9v";
   };
 
   ucdtxt = fetchurl {
-    url = "http://www.unicode.org/Public/11.0.0/ucd/UnicodeData.txt";
-    sha256 = "16b0jzvvzarnlxdvs2izd5ia0ipbd87md143dc6lv6xpdqcs75s9";
+    url = "https://www.unicode.org/Public/13.0.0/ucd/UnicodeData.txt";
+    sha256 = "1fz8fcd23lxyl97ay8h42zvkcgcg8l81b2dm05nklkddr2zzpgxx";
   };
 
   postFixup = ''
     substituteInPlace "$out/bin/.unicode-wrapped" \
       --replace "/usr/share/unicode/UnicodeData.txt" "$ucdtxt"
   '';
 
+  nativeBuildInputs = [ installShellFiles ];
```

Comment:
Final nitpick: could you move `nativeBuildInputs` above postFixup, with a blank line between the ucdtxt attribute and the postFixup attribute?